README.md

```diff
@@ -162,6 +162,17 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+Creating PAM or Native Credentials File (.irodsA)
+-------------------------------------------------
+
+```Two free functions exist for creating encoded authentication files.
+    irods.client_init.write_credentials_with_native_password
+    irods.client_init.write_credentials_with_pam_password
+```
+
+Each takes a cleartext password and creates an appropriately encoded version of it
+as an .irodsA file in the login environment, if none is pre-existing.
+
```

Comment:
As discussed during standup, let's add a usage example here as well.

Reply:
Agree, this is needed.